crates/workspace/src/pane_group.rs

```diff
@@ -720,27 +723,49 @@ impl PaneAxis {
         render_cx: &dyn PaneLeaderDecorator,
         window: &mut Window,
         cx: &mut App,
-    ) -> gpui::AnyElement {
+    ) -> (gpui::AnyElement, bool) {
         debug_assert!(self.members.len() == self.flexes.lock().len());
         let mut active_pane_ix = None;
+        let mut contains_active_pane = false;
+
+        // Track which children are leaf panes (Member::Pane) vs axes (Member::Axis)
+        let mut is_leaf_pane = vec![false; self.members.len()];
 
-        pane_axis(
+        let rendered_children = self
+            .members
+            .iter()
+            .enumerate()
+            .map(|(ix, member)| {
+                // Mark if this is a leaf pane (Member::Pane) or an axis (Member::Axis)
```

Comment:
Self-documenting comment again.

Also, below we seem to do two `matches!(member, Member::Pane(_)` in a row: the 2nd one has extra condition, but nontheless it seems possible to write it into a single `match member` expression instead.